dev-tools/release/prepare_rc.sh

```diff
@@ -228,10 +250,12 @@ mkdir ${FULL_VERSION}
 cp ${GEODE}/geode-assembly/build/distributions/* ${FULL_VERSION}
 cp ${GEODE_EXAMPLES}/build/distributions/* ${FULL_VERSION}
 cp ${GEODE_NATIVE}/build/apache-geode-native-${VERSION}* ${FULL_VERSION}
+cp ${GEODE_BENCHMARKS}/apache-geode-benchmarks-${VERSION}* ${FULL_VERSION}
+rm ${FULL_VERSION}/apache-geode-examples-*.zip*
```

Comment:
Why this change to remove the examples zip?

Reply:
thanks, split this out into #4606